sdk/src/main/scala/com/horizen/params/NetworkParams.scala

```diff
@@ -47,6 +47,7 @@ trait NetworkParams {
   val cswVerificationKeyFilePath: String
   val sidechainCreationVersion: SidechainCreationVersion
   val isCSWEnabled: Boolean
+  val allowUnprotectedTxs: Boolean
```

Comment:
I think we don't need to change NetworkParams and ancestors of it because `allowUnprotectedTxs` is a Node config not a Network config.
In nodeViewHolder we have an access to the `SidechainSettings`